### PR TITLE
userguide: cover install-full and install-conf -v3

### PR DESCRIPTION
**Pull request by comfort619**
Ticket: 6342
Link to [redmine](https://redmine.openinfosecfoundation.org/issues/6342) ticket:

Describe changes:
-Incorporate feedback from last PR.
-Previous PR: #9592

### CODE REVIEW

File: doc/userguide/install.rst [hunk 1]
```diff
@@ -205,8 +231,6 @@ To remove Suricata from your system::
 
     sudo apt-get remove suricata
 
-
-
```
Comment (inashivb) [inline doc/userguide/install.rst:209]: nit: nothing should change on these lines in this commit. No deletions, no additions

**Review by inashivb — CHANGES_REQUESTED** [review]
Hi @comfort619 !
Looking almost ready. One minor change requested from me inline. Thank you. :)

File: doc/userguide/install.rst (hunk 1 above)
Comment (comfort619) [reply doc/userguide/install.rst:209]: Okay, noted

**Review by jufajardini — COMMENTED** [review]
Section organization looking good now ^^

**Comment by inashivb** [comment]
@comfort619 please close any previous PRs when you create a new one. Thanks! :)

**Pull request closed without merging**